### PR TITLE
restructured the repository, fixed issue with PDF generation

### PR DESCRIPTION
**Pull request by svekars**
I completely restructure the ../pages directory, fixed the issue with duplicate TOCs in PDFs. Added links to the PDF files on the title page.

### CODE REVIEW

**Comment by mihgen** [comment]
Add *.swp files to .gitignore, I see .pdf_install.rst.swp to be included now - you need to remove it.

File: pages/old/about-fuel/0040-reference-topologies.rst [hunk 1]
```diff
@@ -0,0 +1,41 @@
+.. raw:: pdf
```
Comment (mihgen) [inline pages/old/about-fuel/0040-reference-topologies.rst]: Why do we need all of the pages/old folder? We don't include it anywhere, do we?

File: install-guide.rst [hunk 2]
```diff
@@ -7,6 +7,7 @@ Installation Guide
```
Comment (mihgen) [inline install-guide.rst:7]: Do we use this file? Looks like we have another one, contents-install.rst for Installation Guide.

File: pages/old/about-fuel/0040-reference-topologies.rst (hunk 1 above)
Comment (svekars) [inline pages/old/about-fuel/0040-reference-topologies.rst]: We don't include the /old folder. But it contains the old structure of the doc. I can remove it, if you are OK with that.

Comment (mihgen) [inline pages/old/about-fuel/0040-reference-topologies.rst]: Sure, let's remove what we don't need anymore. I'll see what's removed, and we can revert it any time anyway.

File: install-guide.rst (hunk 2 above)
Comment (svekars) [inline install-guide.rst:7]: Yes. We use it. The install-guide.rst, user-guide.rst, and so on go to the web-site. The contents-user, contents-install, and so on are the table of contents for the PDFs. If I use  install-guide.rst, user-guide.rst for PDF's table of contents, I see issues in the output.

**Comment by mihgen** [comment]
+2. 
I'll merge it now. We have to fix a lot of regarding the internal content though..

**Pull request merged**